src/dotnet/APIView/APIViewWeb/Pages/Shared/_ReviewBadge.cshtml

```diff
@@ -2,6 +2,11 @@
 @using System
 @using System.Text.RegularExpressions;
 @model ReviewModel;
+@{
+    var packageName = Model.Revisions.LastOrDefault()?.Files.LastOrDefault()?.PackageName;
+    var approver = Model.PackageNameApprovedBy ?? Model.Revisions.LastOrDefault(r => r.IsApproved)?.Approvers?.FirstOrDefault();
+    var nameApprovalTip = approver != null? "Package name was approved by " + approver : "Package name is in approved status";
```

Comment:
In which cases there won't be an approver? and what does that mean?

Reply:
There are many reviews for same package. We show package name as approved when review itself is approved. It's possible that another review is in approved state for same package. We won't have approver name in this case. I will see If I can populate approver name in this case.